docs/Current_Code.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import numpy
+from pyfirmata import Arduino, util
+import time
+from time import sleep
+from pyfirmata import INPUT, OUTPUT, PWM
+import serial
+import re
+import matplotlib.pyplot as plt
+
+class myArduinoBoard(object):
```

Comment:
Classes should always begin with a capital letter.  Why not just ``ArduinoBoard``?